buildspec.yml

```diff
@@ -43,11 +43,11 @@ phases:
       - echo "//npm.pkg.github.com/:_authToken=${GITHUB_ACCESS_TOKEN}" > ~/.npmrc
       - echo Setting environment variables
       - cd .aws
-      - npm install
```

Comment:
why remove npm install here? this repo appears to use cdktf, which may need those packages installed for the synth & terraform to run

Reply:
usually in ci, we run the the package lock command right? i.e what's tested in local is pushed to package-lock and that's run in CI